src/envoy/mixer/integration_test/setup.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"log"
+)
+
+const (
+	// These ports should match with used envoy.conf
+	// Default is using one in this folder.
+	ServerProxyPort = 29090
```

Comment:
It's used in the template. I think the idea is to be able to run config standalone.